content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -777,7 +777,7 @@ Example FlowSchema object to isolate list event requests:
 
 ## {{% heading "whatsnext" %}}
 
-- You can visit flow control [reference doc](/docs/reference/flow-control/) to learn more about troubleshooting.
+- You can visit flow control [reference doc](/docs/reference/debug-cluster/flow-control/) to learn more about troubleshooting.
```

Comment:
Possibly edit the list item:
To learn more about troubleshooting, see the [Flow Control](/docs/reference/debug-cluster/flow-control/) reference.

Reply:
+1